src/client/game/game-reducer.js

```diff
@@ -1,23 +1,58 @@
-import { START_GAME, LEAVE_GAME, SET_LEADER } from './game-actions';
+
+import { START_GAME, LEAVE_GAME, SET_LEADER, GET_CONSONANT,
+         GET_VOWEL, DISABLE_CONSONANT, DISABLE_VOWEL } from './game-actions';
 
 const initialState = {
 	started: false,
+	leader: undefined,
+    letterList: [],
+    disableConsonant: false,
+    disableVowel: false
 };
 
 const game = (state = initialState, action) => {
     switch (action.type) {
-        case START_GAME:
+        case START_GAME: 
         	return Object.assign({}, state, {
         		started: true
         	});
         case LEAVE_GAME:
             return Object.assign({}, state, {
-                started: false
+                started: false,
+                letterList: [],
+                disableConsonant: false,
+                disableVowel: false
             });
         case SET_LEADER:
             return Object.assign({}, state, {
-                leader: action.payload.name
+                leader: action.payload.name,
+                disableConsonant: action.payload.disableConsonant,
+                disableVowel: action.payload.disableVowel
+            });
+        case GET_CONSONANT: {
+            let letterList = Array.from(state.letterList);
+            letterList.push(action.payload);
+            return Object.assign({}, state, {
+                letterList: letterList
+            });
+        }
+        case GET_VOWEL: {
+            let letterList = Array.from(state.letterList);
+            letterList.push(action.payload);
+            return Object.assign({}, state, {
+                letterList: letterList
```

Comment:
`letterList: [...state.letterList, action.payload]` -  you can then remove the variable assignment